### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/views/survey/_evaluationParticipantsView.gsp [hunk 1]
```diff
@@ -1168,6 +1168,7 @@
                     <td></td>
                 </g:else>
             </g:each>
+            <td></td>
```
Comment (croqueGrec09) [inline grails-app/views/survey/_evaluationParticipantsView.gsp:1171]: empty td?

**Review by croqueGrec09 — COMMENTED** [review]
one remark

File: grails-app/views/survey/_evaluationParticipantsView.gsp (hunk 1 above)
Comment (Moetez16) [reply grails-app/views/survey/_evaluationParticipantsView.gsp:1171]: yes. for action area ;)

**Comment by Moetez16** [comment]
see comment

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**